src/main/java/TaskList.java

```diff
@@ -0,0 +1,150 @@
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class TaskList {
+
+    private ArrayList<Item> items;
+
+    public TaskList() {
```

Comment:
would be good to have JavaDoc for your constructor since you have overloaded constructors